test/unit/vl-map-layer.test.html

```diff
@@ -100,31 +116,57 @@
         assert.isTrue(ol.layer.Vector.called);
       });
       
-      test('na het toevoegen van de kaartlaag zal de kaart zoomen naar de extent van de kaartlaag indien de auto-extent geactiveerd werd', () => {
+      test('na het toevoegen van de kaartlaag zal de kaart zoomen naar de extent van de kaartlaag indien de auto-extent geactiveerd werd', async () => {
         let kaartElement = fixture('vl-map-fixture');
-        let kaartlaagElement = kaartElement.querySelector('vl-map-layer');
-        assert.notDeepEqual(kaartElement.map.getView().getCenter(), [147055, 197908]);
-        
+        await awaitUntil(() => {
+          return !isEqual(kaartElement.map.getView().getCenter(), [147055, 197908]);
+        });
```

Comment:
kunt ge hier niet beter awaiten until er iets wel waar is ipv iets wat niet waar is?

Reply:
Yes, dat kan. Ik had for now gewoon een awaitUntil geschreven rond de bestaande conditie. Zal dat aanpassen.